packages/terra-modal/src/modal.scss

```diff
@@ -0,0 +1,81 @@
+@import 'node_modules/terra-legacy-theme/src/terra-legacy-theme';
```

Comment:
We cannot import a theme in any package because we need to be able to switch it out for a different theme. The webpack configuration. 

Currently, in the webpack configuration a consumer will specify the theme: https://github.com/cerner/terra-ui/blob/master/webpack.config.js#L42-L44

This theme file will be loaded before any other scss file.